stargate-core/app/com/salesforce/mce/stargate/controllers/McSsoController.scala

```diff
@@ -52,8 +52,14 @@ class McSsoController @Inject() (
       id <- request.session.data.get("id")
     } yield (mid.toLong, userId.toLong, id)
 
-    data.fold(Future.successful(Ok.withNewSession)) { case (mid, userId, id) =>
-      sessionTrackingService.destroy(mid, userId, id).map(_ => Ok.withNewSession)
+    val result = Ok.withCookies(Cookie(
+      name = AppConfig.cookieName,
+      value = "",
+      secure = AppConfig.secure,
+      sameSite = SameSite.parse(AppConfig.sameSite)
```

Comment:
I thought the bug is that we should always override `sameSite` to `None` regardless of the setting?

Reply:
I think we should read from config, since this is a library and applications should define what sameSite value is for them